client/src/api/databases/index.js

```diff
@@ -0,0 +1,34 @@
+import api from '../../api'
+let model = 'databases'
+export default {
+  get: () => {
```

Comment:
=> Add parameter id.
=> In method check id is undefined then return all recorder if id is exist then return only one.